File: AtlasWeb/Core/wwwroot/app/NebulaClient/StorageManager/DefinitionManagement/TTInterfaceMemberDef.ts
import { Guid } from "../../Mscorlib/Guid";
import { TTInterfaceDef } from "./TTInterfaceDef";

export class TTInterfaceMemberDef {
    public InterfaceDef: TTInterfaceDef;
    public InterfaceDefID: Guid;
    public Name: string;
    public Description: string;
    public CodeType: string;
    public IsReadOnly: boolean;
    public ToString(): string {
        return null;
    }
}

File: ATLASDEV/AtlasModel/Enterprise/TTInterfaceMemberDef.cs
using System;

namespace AtlasModel.Enterprise
{
    public partial class TTInterfaceMemberDef
    {
        public Guid InterfaceDefId { get; set; }
        public string Name { get; set; }
        public string CodeType { get; set; }
        public bool IsReadOnly { get; set; }
        public short CheckOutStatus { get; set; }
        public string Description { get; set; }
    }
}